libwallet/internal/vsp/feepayment.go

```diff
@@ -92,21 +89,23 @@ type feePayment struct {
 	feeAddr       stdaddr.Address
 	feeHash       chainhash.Hash
 	feeTx         *wire.MsgTx
-	state         state
+	state         State
 	err           error
 
 	timerMu sync.Mutex
 	timer   *time.Timer
+
+	params *chaincfg.Params
 }
 
-type state uint32
+type State uint32
```

Comment:
I assume `State` is not used outside the `internal/vsp` pkg, although I didn't take notice of it's use elsewhere. Please confirm.